### ISSUE TITLE
Clarification question regarding PME vs LJPME

### ISSUE THREAD

**Issue by varun-go** [openmm/openmm#4621]
My question is whether the energy computed using the NonbondedForce should differ when using 1) LJPME or 2) PME with dispersion corrections included.

When computing the energy for a single frame using the NonbondedForce, I found that the resultant energy differs between the above mentioned methods. I could not understand the difference in the documentation. I found a similar question in issue #2224, but I was not able to clarify my question from that. Any help would be appreciated. Thank you!

**Comment by peastman** [comment]
Yes, they're expected to be different.  LJPME sums all the attractive $1/r^6$ interactions out to infinity.  The dispersion correction just tries to approximate the interactions beyond the cutoff by assuming a uniform distribution of particles.  It's a pretty good approximation for isotropic systems, less good for non-isotropic ones.  But it's just an approximation in either case.

The two methods are described in http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#lennard-jones-interaction and http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#lennard-jones-interaction-with-particle-mesh-ewald.
